File: recursion/decToBin.cpp
#include <iostream>

void decToBin(int decValue){
    if(decValue > 0){
    int remainder = decValue % 2;
    int quotient = (decValue - remainder) / 2;
    decToBin(quotient);
    std::cout << remainder;
    }
}

int main(){

    decToBin(2);

    return 0;
}

File: Elvis/recursion/decToBin.js
let decToBin = (dev_value) => {
    let quotient, remainder;
    remainder = (dev_value % 2);
    quotient = (dev_value - remainder) / 2;
    remainder.toString();
    if(dev_value === 1) return remainder;
    else return `${decToBin(quotient)}${remainder}`;
}